src/server/worker/worker.go

```diff
@@ -119,7 +119,7 @@ func (w *Worker) worker(env serviceenv.ServiceEnv) {
 			logger.Logf("worker failed, retrying in %v:\n%s", d, err)
 		}
 		return nil
-	})
+	}) //nolint:errcheck
```

Comment:
Since this is a retry until cancel, it can eventually return an error, right? Is it worth checking it?

Reply:
I think the guarantee is that the error is "context canceled" and that the context is pctx.Background() so not possible to cancel.  It doesn't have to be pctx.Background() of course, but it is in this case.

Reply:
The justification for `nolint` should say that, though.

Reply:
> It doesn't have to be pctx.Background() of course, but it is in this case.

It *does* look like nothing in the dynamic contour ends up with a cancellable context … which raises the question of why this is calling `RetryUntilCancel`.